servers/jax-rs-testextension/src/main/java/org/projectnessie/jaxrs/ext/NessieJaxRsExtension.java

```diff
@@ -216,6 +216,7 @@ void setAccessChecker(Function<AccessContext, BatchAccessChecker> accessChecker)
     public EnvHolder(Extension versionStoreExtension) throws Exception {
       weld = new Weld();
       // Let Weld scan all the resources to discover injection points and dependencies
+      weld.addPackages(true, RestConfigResource.class);
```

Comment:
Shouldn't this be in the configure() method below?

Reply:
discussion in https://github.com/projectnessie/nessie/pull/5828#discussion_r1066834290